crates/release_plz/src/main.rs

```diff
@@ -1,110 +1,38 @@
-mod args;
 mod changelog_config;
+mod commands;
 mod config;
-mod generate_schema;
-pub mod init;
 mod log;
-mod update_checker;
 
-use args::{config_command::ConfigCommand as _, OutputType};
-use clap::Parser;
-use config::Config;
-use release_plz_core::{ReleasePrRequest, ReleaseRequest, UpdateRequest};
-use serde::Serialize;
+use clap::{
+    builder::{styling::AnsiColor, Styles},
+    Parser,
+};
+use commands::Command;
 use tracing::error;
 
-use crate::args::{manifest_command::ManifestCommand as _, CliArgs, Command};
+const MAIN_COLOR: AnsiColor = AnsiColor::Red;
+const SECONDARY_COLOR: AnsiColor = AnsiColor::Yellow;
+const HELP_STYLES: Styles = Styles::styled()
+    .header(MAIN_COLOR.on_default().bold())
+    .usage(MAIN_COLOR.on_default().bold())
+    .placeholder(SECONDARY_COLOR.on_default())
+    .literal(SECONDARY_COLOR.on_default());
+
+#[derive(clap::Parser, Debug)]
+#[command( version, author, styles = HELP_STYLES)]
+pub struct CliArgs {
```

Comment:
Move this into main.rs from args as it's now fairly small and it helps give context to the main function which is also rather anemic now that the command parsing / matching has been pushed down. It could still live in an args.rs perhaps, but it makes a bit of sense from a reading code perspective to have this here. (guiding principal, keep small related things together)

Reply:
I would prefer to have this in its own file 👍